lib/src/attributes/variants/variants.utils.dart

```diff
@@ -132,4 +130,28 @@ class VariantUtils {
       },
     );
   }
+
+  /// Short Utils: active
+  static Variant<T> active<T extends Attribute>() {
+    return Variant<T>(SystemVariants.active.value);
+  }
+
+  /// Short Utils: inactive
+  static Variant<T> inactive<T extends Attribute>() {
+    return Variant<T>(SystemVariants.inactive.value);
+  }
+
+  /// Short Utils: title
+  static Variant<T> title<T extends Attribute>() {
```

Comment:
Same for this variant. I believe any variant for business logic implementation that is not generic for use in the primitives should not be here